ddtrace/constants.py

```diff
@@ -44,7 +48,12 @@
 MANUAL_KEEP_KEY = "manual.keep"
 
 
-ERROR_MSG = "error.msg"  # a string representing the error message
+ERROR_MESSAGE = "error.message"  # a string representing the error message
+ERROR_MSG = ERROR_MESSAGE
+removals.removed_constant(
```

Comment:
... does this function/helper exist? I cannot find it in the debtcollector vendored code

Reply:
See below